modules/3d-tiles/src/lib/tileset/helpers/bounding-volume.js

```diff
@@ -65,21 +65,22 @@ export function createBoundingVolume(boundingVolumeHeader, transform, result) {
 
 function createBox(box, transform, result) {
   const center = new Vector3(box[0], box[1], box[2]);
-  let halfAxes = new Matrix3(box.slice(3, box.length));
-
   transform.transform(center, center);
 
-  halfAxes = new Matrix3(
-    transform[0],
-    transform[1],
-    transform[2],
-    transform[4],
-    transform[5],
-    transform[6],
-    transform[8],
-    transform[9],
-    transform[10]
-  ).multiplyRight(halfAxes);
+  const xAxis = transform.transformAsVector(box.slice(3, 6));
```

Comment:
Nits: 
- As below use `Matrix4.getColumn`? That would make the code here more self-documenting. 
- Perhaps a line of comment about what is happening?

Reply:
`box` is not a matrix, but an array defined by the tile schema. I'll add a comment.

Reply:
Yes that sounds right. I suspect this corresponds exactly to the layout of the math.gl OrientedBoundingBox class..

Which should have a toSphere method...